### PR TITLE
feat: Celery mail

### PR DESCRIPTION
**Pull request by jaspnas**
Send email using celery workers

### CODE REVIEW

File: core/utils.py [hunk 1]
```diff
@@ -28,4 +31,12 @@ def validate_captcha(response: str) -> bool:
...
\ No newline at end of file
+    return res.json().get('success', False)
+
+
+@shared_task
+def send_email_task(*args, **kwargs) -> None:
+    try:
+        send_mail(*args, **kwargs)
+    except SMTPException:
+        print(f"Failed sending email to: {args[3] or kwargs.get('to', '')}")
```
Comment (theoabw) [inline core/utils.py]: Might be better to use logger.error() instead?

File: members/templates/registration/password_reset_email.html [hunk 2]
```diff
@@ -4,11 +4,11 @@
 
 {{ protocol }}://{{ domain }}{% url 'password_reset_confirm' uidb64=uid token=token %}
```
Comment (theoabw) [inline members/templates/registration/password_reset_email.html:5]: Might wanna try `<a href="{{ protocol }}://{{ domain }}{% url 'password_reset_confirm' uidb64=uid token=token %}">{{ protocol }}://{{ domain }}{% url 'password_reset_confirm' uidb64=uid token=token %}</a>` to ensure clickability?

**Review by Theoabw — CHANGES_REQUESTED** [review]

File: core/utils.py (hunk 1 above)
Comment (jaspnas) [reply core/utils.py]: Good catch

File: members/templates/registration/password_reset_email.html (hunk 2 above)
Comment (jaspnas) [reply members/templates/registration/password_reset_email.html:5]: Not going to change it in this PR, as it seems to work for now without it.

**Review by Theoabw — APPROVED** [review]

**Pull request merged**